pkg/ibmvpc/parser.go

```diff
@@ -576,9 +577,9 @@ func getVPCconfig(rc *datamodel.ResourcesContainerModel,
 		newVPCConfig := NewEmptyVPCConfig()
 		newVPCConfig.UIDToResource[vpcNodeSet.ResourceUID] = vpcNodeSet
 		newVPCConfig.VPC = vpcNodeSet
-		res[vpcNodeSet.ResourceUID] = newVPCConfig
+		res.AddConfig(vpcNodeSet.ResourceUID, newVPCConfig)
```

Comment:
Seems that you parse only `configs;` what about `toCompareConfigs` ?
(the tests have a workaround and so the error was not caught by them)
Or am I missing something?

Reply:
in general, I stick  to the old code behavior. and did not change it.
I did not dive into it, but I think that what you are missing that the parser is called twice. so this code does not know if it parse a config to compare or not

Reply:
Here I disagree. Using the functionality of the 1st config file to parse the 2nd is ugly and confusing. I think it should be changed within this PR. Since we seem to disagree let @adisos decide.

Reply:
this PR is only about editing the multipleVPCConfigs struct.
if you think that there is a problem with the flow of the parser/ main/testing. please address it to a new issue.